core/common/assets/js/api/modules/command-data.js

```diff
@@ -142,15 +142,15 @@ export default class CommandData extends CommandBase {
 	 * @param [args={}]
 	 * @returns {{}} filtered result
 	 */
-	applyAfterDelete( data, args = {} ) {
+	applyAfterDelete( data, args = {} ) {// eslint-disable-line no-unused-vars
```

Comment:
We should find a better solution for that, instead of `eslint-disable-line no-unused-vars` everywhere

Reply:
Maybe we can use this:
````
/* eslint-disable no-unused-vars */

// ... your code here with unused vars...

/* eslint-enable no-unused-vars */
````

when we sure what we do?

Reply:
By the way: its not everywhere its in base classes + tests